go/protocol/chat1/common.go

```diff
@@ -234,6 +234,18 @@ type MessageClientHeader struct {
 	OutboxInfo   *OutboxInfo              `codec:"outboxInfo,omitempty" json:"outboxInfo,omitempty"`
 }
 
+type MessageClientHeaderVerified struct {
+	Conv         ConversationIDTriple     `codec:"conv" json:"conv"`
+	TlfName      string                   `codec:"tlfName" json:"tlfName"`
+	TlfPublic    bool                     `codec:"tlfPublic" json:"tlfPublic"`
+	MessageType  MessageType              `codec:"messageType" json:"messageType"`
+	Prev         []MessagePreviousPointer `codec:"prev" json:"prev"`
+	Sender       gregor1.UID              `codec:"sender" json:"sender"`
+	SenderDevice gregor1.DeviceID         `codec:"senderDevice" json:"senderDevice"`
```

Comment:
Cool. Fyi I'm doing the same stuff in https://github.com/keybase/client/pull/6077/files#diff-d1325d9a259c7d25092218f3fa0661d4R554 so I'll take care of unifying those.